### PR TITLE
design/1-auto-score.md: add proposal "Automatically score Vulcan assets"

### PR DESCRIPTION
**Pull request by jroimartin**
Add proposal "Automatically score Vulcan assets".

Discussion at https://github.com/adevinta/graph-proposal/issues/1.

### CODE REVIEW

File: design/1-auto-score.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+Ideally, assets should be scored automatically.
+Now with the Security Graph we have a source of objective data that allows to
+automatically characterized assets.
+For instance, the Blast Radius Score, calculated using the AWS data contained
+in the Security Graph, measures the number of systems that could be potentially
+affected by a successful attack.
+Of course, there are other useful sources of information and many more
+interesting scores.
+
+The final solution must combine manual and automated scores in a single metric
```
Comment (julianvilas) [inline design/1-auto-score.md]: Should we mention that the "final solution" is not the outcome of this proposal?

File: design/1-auto-score.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+The proposed approach, allows to use the `AutoScore` field to store automatic
+scores derived from manual scores.
+This way a modification in the ROLFP of an asset could trigger the update of a
+specific automatic score.
+This could even imply querying the Security Graph to merge multiple ROLFP
+scores together, potentially replacing the `Scope` functionality of the current
+scoring model.
+
+Finally, it would be the combination of all these automatic scores what is used
+to take actions.
```
Comment (julianvilas) [inline design/1-auto-score.md:280]: Should we reinforce that this is not part of the current proposal?

File: design/1-auto-score.md (hunk 1 above)
Comment (jroimartin) [reply design/1-auto-score.md]: Given that this is the background and it is already mentioned both in the abstract and later in the Rationale (see commit ebf17ea), I didn't added another instance here. However, I reworded it to be less specific.

File: design/1-auto-score.md (hunk 2 above)
Comment (jroimartin) [reply design/1-auto-score.md:280]: Done in ebf17ea.

**Review by julianvilas — APPROVED** [review]

**Comment by jroimartin** [comment]
Archived repository.

**Pull request closed without merging**